tests/unit/bokeh/models/test_glyphs.py

```diff
@@ -55,19 +55,27 @@
 
 # Module under test
 from bokeh.models.glyphs import ( # isort:skip
```

Comment:
Given the changes, then this can be removed now:
```suggestion
from bokeh.models.glyphs import (
```